src/quo2/components/selectors/react_selector_add/view.cljs

```diff
@@ -0,0 +1,44 @@
+(ns quo2.components.selectors.react-selector-add.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]
+            [react-native.core :as rn]))
+
+(defn- get-color
+  [pinned-colors default-colors pinned? theme]
+  (apply colors/theme-colors
+         (conj (if pinned? pinned-colors default-colors)
+               theme)))
+
+(defn- container-style
+  [pinned? theme]
+  (let [pinned-cols  [colors/neutral-80-opa-5 colors/white-opa-5]
+        default-cols [colors/neutral-20 colors/neutral-70]
+        border-color (get-color pinned-cols
+                                default-cols
+                                pinned?
+                                theme)]
+    {:justify-content    :center
+     :align-items        :center
+     :padding-horizontal 7
+     :border-radius      8
+     :border-width       1
+     :height             24
+     :border-color       border-color}))
+
```

Comment:
@clauxx 
as @J-Son89 said, this piece of code seems more complex than needed.

I suggest removing that `get-color` function and simplify that `container-style` fn. we can just use an `if` which is very simple:

```clojure
(defn- container-style
  [pinned? theme]
  (let [border-color (if pinned?
                       (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5 theme)
                       (colors/theme-colors colors/neutral-20 colors/neutral-70 theme))]
    {:justify-content    :center
     :align-items        :center
     :padding-horizontal 7
     :border-radius      8
     :border-width       1
     :height             24
     :border-color       border-color}))
```